public/home.css

```diff
@@ -0,0 +1,4 @@
+* {
+  font-family: 'Droid Sans', sans-serif;
```

Comment:
Universal application of "Droid Sans" google font and a dark green background-color.